requests/models.py

```diff
@@ -672,6 +672,7 @@ def __iter__(self):
 
     @property
     def ok(self):
+        """Returns true if :attr:`status_code` is 'OK'."""
```

Comment:
That's actually not true. This returns `True` if `status_code < 400`. So this will return True for `2xx`, `3xx` responses. This isn't a `status_code == 200` check.